embed_video/tests/backends/tests_wistia.py

```diff
@@ -17,7 +17,7 @@ class WistiaBackendTestCase(BackendTestMixin, TestCase):
 
     def test_wistia_get_info_exception(self):
         with self.assertRaises(VideoDoesntExistException):
-            backend = WistiaBackend('http://support.wistia.com/123')
+            backend = WistiaBackend('https://support.wistia.com/123')
             backend.get_info()
```

Comment:
I noticed my tests didnt have https in them either, so I changed those just to be consistent.